docs/chapters/02_getting-started.md

```diff
@@ -600,25 +601,31 @@ You can also clean the compiled code by running:
 boost clean
 ```
 
-Now, let's deploy our application to the cloud to see it working. It is as simple as running
+Now, let's run our application to see it working. It is as simple as running:
+
+```bash
+boost start -e local
+```
+
+Or, we can deploy our application to the cloud with no additional changes by running
 the deploy command:
 
 ```bash
 boost deploy -e production
 ```
 
-And here it comes the Booster magic! ✨ When running the deploy command, Booster will handle the creation of all the resources, *like Lambdas, API Gateway,* and the "glue" between them; *permissions, events, triggers, etc.* It even creates a fully functional GraphQL API!
+This is the Booster magic! ✨ When running the start or the deploy commands, Booster will handle the creation of all the resources, _like Lambdas, API Gateway,_ and the "glue" between them; _permissions, events, triggers, etc._ It even creates a fully functional GraphQL API!
 
 > [!NOTE] Deploy command automatically builds the project for you before performing updates in the cloud provider,
-so, build command it's not required beforehand.
+> so, build command it's not required beforehand.
 
 > With `-e production` we are specifying which environment we want to deploy. We'll talk about them later.
 
 > [!TIP] If at this point you still don’t believe everything is done, feel free to check in your provider’s console. You should see, as in the AWS example below, that the stack and all the services are up and running! It will be the same for other providers. 🚀
 
 ![resources](../img/aws-resources.png)
 
-It will take a couple of minutes to deploy all the resources. Once finished, you will see
+When deploying, it will take a couple of minutes to deploy all the resources. Once finished, you will see
```

Comment:
```suggestion
When deploying, it will take a couple of minutes to provide all resources for you. Once finished, you will see
```
The verb "deploy" is written twice, but I'm not 100% sure about which word we should use 🤔.